website/blog/2021-08-12-tungsten.md

```diff
@@ -0,0 +1,343 @@
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Add support for `textDocument/selectionRange`
+- [Scala 3] `Inferred Type` code action and other improvements
+- Formatter - `runner.dialect` support and `.scalafmt.conf` update suggestions
+- Dependency sources - search symbol references in the workspace
+
+## Add support for `textDocument/selectionRange`
+
+Due to the great work by [@ckipp01](https://github.com/ckipp01) Metals now implements the `textDocument/selectionRange` [LSP-method](https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_selectionRange).
+It allows you to easily select an expression and expand/shrink it using shortcut combinations:
+
+
+![gif1](https://user-images.githubusercontent.com/13974112/125335989-7a5f8780-e34d-11eb-911f-42f851478737.gif)
+
+Default keybindings in VSCode:
+<table>
+  <tbody style={{"font-size": "13px"}}>
+  <tr>
+    <td></td>
+    <td>Win/Linux</td>
+    <td>macOS</td>
+  </tr>
+  <tr>
+    <td>Expand</td>
+    <td>Shift+Alt+Right</td>
+    <td>⌃⇧⌘→</td>
+  </tr>
+  <tr>
+    <td>Shrink</td>
+    <td>Shift+Alt+Left</td>
+    <td>⌃⇧⌘←</td>
+  </tr>
+  </tbody>
+</table>
+
+## [Scala3] `Inferred Type` code action and other improvements
+
+We're hard at work ensuring that the same feature-set for Scala 2 is available for Scala 3.
+This release adds Inferred Type code action support.
+
+![gif2](https://i.imgur.com/GJGFOOy.gif)
+
+In addition to other small fixes, it's worth mentioning that the `Go to Definition` functionality was improved so that in situations where there are compilation issues, you're now able to better navigate to the definition site.
+Previously, it worked improperly in case if there were compile errors. 
+
+## Formatter - `runner.dialect` support and `.scalafmt.conf` update suggestions.
+
+In order to support different Scala dialects scalafmt added a [`runner.dialect` setting](https://scalameta.org/scalafmt/docs/configuration.html#scala-3) to support Scala 3.
+Now Metals knows about it and can correctly generate and even automatically upgrade `.scalafmt.conf`
+
+## Dependency sources - search symbol references in the workspace
+
+Thanks to [@Z1kkurat](https://github.com/Z1kkurat) "Find References" now also works from within your dependency sources to include references from all workspace sources.
+Previously, it included only local occurrences in the file.
+
+![gif3](https://i.imgur.com/myHPDjP.gif)
+
+
+## Contributors
+
+Big thanks to everybody who contributed to this release or reported an issue!
+
+```
+$ git shortlog -sn --no-merges v0.10.4..v0.10.5
+22	Scala Steward
+22	ckipp01
+18	Vadim Chelyshov
+17	Tomasz Godzik
+ 2	Adrien Piquerez
+ 2	Luigi Frunzio
+ 2	dependabot[bot]
```

Comment:
Let's remove dependabot and Scala Steward, they're fun, but still just robots :D